splinterd/src/config/default.rs

```diff
@@ -14,8 +14,9 @@
 
 use crate::config::{ConfigError, ConfigSource, PartialConfig, PartialConfigBuilder};
 
-const DEFAULT_CERT_DIR: &str = "/etc/splinter/certs/";
-const DEFAULT_STATE_DIR: &str = "/var/lib/splinter/";
+pub const CONFIG_DIR: &str = "/etc/splinter";
```

Comment:
This hsould be DEFAULT_CONFIG_DIR.

The pub should be removed here and on the following two lines.